src/lib/spinel/radio_spinel.cpp

```diff
@@ -51,6 +51,35 @@
 namespace ot {
 namespace Spinel {
 
+char RadioSpinel::sVersion[kVersionStringSize] = "";
+
+otExtAddress RadioSpinel::sIeeeEui64;
+
+bool RadioSpinel::sIsReady = false; ///< NCP ready.
+
+bool RadioSpinel::sSupportsLogStream =
+    false; ///< RCP supports `LOG_STREAM` property with OpenThread log meta-data format.
+
+bool RadioSpinel::sSupportsResetToBootloader = false; ///< RCP supports resetting into bootloader mode.
+
+otRadioCaps RadioSpinel::sRadioCaps = OT_RADIO_CAPS_NONE;
```

Comment:
Same with `RadioCaps`? Do we assume all IID instances on RCP provides the same set of caps?

Reply:
For now I assumed all instances to be equal. This can be changed in the future if we find a case where there is a need to have different caps in different instances. Such change will probably also require some "IID search" mechanism on the host to find IID that suits the protocol needs.